introduction.md

```diff
@@ -0,0 +1,2 @@
+Hello, I'm Dave, living in Phnom Penh, trying to make sense of GitHub.  It is arcane!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.